src/component/share/index.tsx

```diff
@@ -22,6 +22,16 @@ import { IInvite } from "module/interface/invite";
 import { TimelineContext } from "context/TimelineContext";
 import { useUpdateTimeline } from "module/hook/timeline";
 import Select from "component/Select";
+import { toast } from "react-toastify";
+
+function copyURL() {
+  const el = document.createElement("input");
+  el.value = window.location.href.split("/members")[0];
```

Comment:
This is just the placeholder, will confirm with @tfussell to get the real Share URL

Reply:
For a venture this should be `https://{window.location.origin}/{ventureId}/feed` and for a timeline it should be `https://{window.location.origin}/{ventureId}/{timelineId}/feed`.

Reply:
@tfussell done, please check again